codespell_lib/data/dictionary.txt

```diff
@@ -16564,7 +16577,8 @@ lpatform->platform
 lsat->last
 lsit->list
 lsits->lists
-lukid->likud
+lukid->lucid, likud,
```

Comment:
Maybe also add `lusid->lucid` ?

Reply:
Only use of lusid look intentional 

https://www.grep.app/search?q=lusid&words=true

Reply:
Fair enough, I was just thinking that somebody who is bad at spelling, might misspell it that way. But perhaps I was over-reaching.